### PR TITLE
Add docs website

### PR DESCRIPTION
**Pull request by osdnk**

### CODE REVIEW

**Comment by adamczyk777** [comment]
The only thing I see we have to do is to rewrite `README.md` so that it looks more like in Gesture Handler Repo, + need to add it to github pages

**Comment by osdnk** [comment]
I think it will be cool if we split nodes for separated page and then group them in menu (see rn-paper)

File: docs/component-docs.config.js [hunk 1]
```diff
@@ -3,6 +3,18 @@
 import path from 'path';
 import fs from 'fs';
 
+Object.defineProperty(Array.prototype, 'flat', {
```
Comment (osdnk) [inline docs/component-docs.config.js]: Can't we do it a bit better?

**Comment by adamczyk777** [comment]
Hey @osdnk I thought it might be cool to actually ship this thing xD do you think is ready? I think I can handle it as you are absent now 😄

**Comment by osdnk** [comment]
xD

**Pull request merged**